lib/shoryuken/polling/strict_priority.rb

```diff
@@ -73,11 +78,22 @@ def queue_paused?(queue)
         @paused_until[queue] > Time.now
       end
 
-      def pause(queue)
+      def delay_pause(queue)
         return unless delay > 0
 
         @paused_until[queue] = Time.now + delay
-        logger.debug "Paused #{queue}"
+        logger.debug "Paused #{queue}, no messages found"
```

Comment:
remove logging before merge?

Reply:
I figured now that there's multiple reasons a queue could be paused, I should be more specific in the messages. These are DEBUG-level logs so they won't appear unless you opt into more verbose logs.